scala/ppml/demo/deploy-local-spark-sgx.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+export ENCLAVE_KEY_PATH=YOUR_LOCAL_ENCLAVE_KEY_PATH
+export DATA_PATH=YOUR_LOCAL_DATA_PATH
+export KEYS_PATH=YOUR_LOCAL_KEYS_PATH
+export LOCAL_IP=YOUR_LOCAL_IP
+export DOCKER_IMAGE=intelanalytics/bigdl-ppml-trusted-big-data-ml-python-graphene:0.14.0-SNAPSHOT
```

Comment:
Image name should be the same

Reply:
Rename to `intelanalytics/bigdl-ppml-trusted-big-data-fl-scala-graphene:flDemo`. Can I use this name?